### PR TITLE
Create project

### PR DESCRIPTION
**Pull request by EuhenySiniy**

### CODE REVIEW

File: src/com/company/mycallableandfuture/MyCallableAndFuture.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            allFutures.add(future);
+        }
+
+        for (int i = 0; i < COUNT_OF_ITERATION; i++) {
+            Future<Integer> future = allFutures.get(i);
+            System.out.println("Future equal: " + future.get());
+        }
+    }
+
+    static class MyCallable implements Callable<Integer> {
```
Comment (Taniann) [inline src/com/company/mycallableandfuture/MyCallableAndFuture.java]: better move this class to the separate file

File: src/com/company/myexecutorservice/MyExecutorService.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public static void main(String[] args) {
+        ExecutorService executorService = Executors.newFixedThreadPool(THREADS);
+        for (int i = 0; i < ITERATIONS; i++) {
+            executorService.execute(new Value(i));
+        }
+        executorService.shutdown();
+    }
+
+    static class Value implements Runnable {
```
Comment (Taniann) [inline src/com/company/myexecutorservice/MyExecutorService.java]: the same

File: src/com/company/mysheludedfuture/MySheludedFuture.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    public static void main(String[] args) {
+        ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
+        ScheduledFuture<String> firstScheduledFuture = scheduledExecutorService.schedule(new FirstThread(), 3, TimeUnit.SECONDS);
+        ScheduledFuture<String> secondScheduledFuture = scheduledExecutorService.schedule(new SecondThread(), 6, TimeUnit.SECONDS);
+
+        System.out.println(firstScheduledFuture.getDelay(TimeUnit.SECONDS) + 1 + "seconds");
+        System.out.println(secondScheduledFuture.getDelay(TimeUnit.SECONDS) + 1 + "seconds");
+    }
+
+    static class FirstThread implements Callable<String> {
```
Comment (Taniann) [inline src/com/company/mysheludedfuture/MySheludedFuture.java]: better move this class from app class

File: src/com/company/mycallableandfuture/MyCallableAndFuture.java (hunk 1 above)
Comment (EuhenySiniy) [reply src/com/company/mycallableandfuture/MyCallableAndFuture.java]: created

File: src/com/company/myexecutorservice/MyExecutorService.java (hunk 2 above)
Comment (EuhenySiniy) [reply src/com/company/myexecutorservice/MyExecutorService.java]: created

File: src/com/company/mysheludedfuture/MySheludedFuture.java (hunk 3 above)
Comment (EuhenySiniy) [reply src/com/company/mysheludedfuture/MySheludedFuture.java]: resolved

**Comment by EuhenySiniy** [comment]
Sorry for waiting, yesterday there was no access to the internet

**Pull request merged**